website_blog_layout_options/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Layout Options and Extensions for Blog",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Website",
+    "summary": "Adds options and layout for the blog",
+    "depends": [
+        'website_blog'
+    ],
+    "data": [
```

Comment:
@gfcapalbo IMO `data` sequence should [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)